config/routes.rb

```diff
@@ -353,6 +353,7 @@
           end
 
           collection do
+            patch '/bulk_auto_accept', to: 'registrations#bulk_auto_accept', as: :bulk_auto_accept
```

Comment:
Nit: You don't need the leading slash. Rails will use slashes as "glue" for nested routes by default (note that no other route in the `collection` specifies leading slashes)

Reply:
Also, I don' think you need to specify the `as:` alias. (Because the route param and the controller action are both already called exactly `bulk_auto_accept` anyways). If in doubt, use `bin/rails routes` to let Rails show you a full list including all helper names